intrahospital_api/update_demographics.py

```diff
@@ -194,6 +195,69 @@ def update_if_changed(instance, update_dict):
         instance.save()
 
 
+def get_mrn_and_date_from_merge_comment(merge_comment):
+    """
+    Takes in an merge comment e.g.
+    " Merged with MRN 123456 Oct 18 2014 11:03AM  Merged with MRN 234567 on Oct 22 2013  4:44PM"
+    returns a list of (MRN, datetime_merged,)
+    """
+    regex = r'Merged with MRN (?P<mrn>\w*\d*) on (?P<month>\w\w\w)\s(?P<day>[\s|\d]\d) (?P<year>\d\d\d\d)\s(?P<HHMM>[\s|\d]\d:\d\d)(?P<AMPM>[A|P]M)'
+    found = list(set(re.findall(regex, merge_comment)))
+    result = []
+    for match in found:
+        mrn = match[0]
+        date_str = f"{match[2]} {match[1]} {match[3]} {match[4]}{match[5]}"
+        merge_dt = datetime.datetime.strptime(date_str, "%d %b %Y %I:%M%p")
+        result.append((mrn, timezone.make_aware(merge_dt),))
+    # return by merged date
+    return sorted(result, key=lambda x: x[1], reverse=True)
+
+
+def get_related_rows_for_mrn(mrn):
```

Comment:
Is this really what you want? Is there a reason I need more than `fn(mrn)` ... `return primary, [merged_mrn, merged_mrn,...]` ?